src/pages/balance.tsx

```diff
@@ -2,18 +2,10 @@ import { useState } from "react"
 
 import ConnectorModal from "@components/ConnectorModal"
 import { useToken } from "@hooks/useToken"
-import {
-  useAccount,
-  useChainId,
-  useIsActive,
-  useProvider,
-  useSwitchChain,
-} from "@hooks/useWeb3React"
+import { useSwitchChain, useWeb3Context } from "@hooks/useWeb3React"
 
 export default function Home() {
-  const isActive = useIsActive()
-  const account = useAccount()
-  const chainId = useChainId()
+  const { active, account, chainId } = useWeb3Context()
```

Comment:
Hey @agcty. Thanks for the work you're doing on the `web3-react` project. Also, great web3 starter plate.  

I know this PR is not yet done. Was just wondering... Initially had a similar structure like your web3-starter. Eventually ended up wrapping the whole _app with a Provider that consumes priorities hooks. Guess that you are also taking that path, right? Instead of consuming `useAccount`, just get the state directly from the web3 context.

Reply:
Yes exactly, the context is now officially part of web3 v8 again so that should make things even easier!